src/Data/Cursor/AbstractCursor.php

```diff
@@ -55,13 +56,22 @@ abstract class AbstractCursor {
 	 */
 	public $query_vars = [];
 
+	/**
+	 * Stores SQL statement alias for the ID column applied to the cutoff
+	 *
+	 * @var string
+	 */
+	protected $id_key = '';
+
 	/**
 	 * The constructor
 	 *
-	 * @param array $query_vars
-	 * @param string|null $cursor
+	 * @param array       $query_vars         Query variable for the query to be executed.
+	 * @param string      $id_key             .
+	 * @param string|null $cursor             Cursor type. Either 'after' or 'before'.
+	 * @param array       $initial_threshold  
```

Comment:
Add a comment explaining what this is please 🙏

Reply:
🤦🏿‍♂️Crap I didn't it. I'll put it back.